### ISSUE TITLE
Homepage redesign

### ISSUE THREAD

**Issue by mverzilli** [flattenthecurve/guide#213]
Let's use this issue to centralize all discussions and links to relevant content to put in the next version of the Homepage. @jackiedelc, @emersonthis, @weltyj  and myself will lead this effort. 

Content is currently based in the following doc/s:

-https://docs.google.com/document/d/1m4HP1jIShnNqvxD5V7Q9MYbhU7pqSW1JAK89Q6WQ5zU/edit

Things we need to account for (feel free to suggest others):

- [ ] Where the current homepage content should go.
- [ ] GX design to better convey the new message.
- [ ] How this impacts versions of the site in other languages. 

Let's ship this! 💪

**Comment by emersonthis** [comment]
Progress on this: https://github.com/flattenthecurve/guide/tree/issue-213

**Comment by mverzilli** [comment]
Awesome! Could you open a PR and label it with "WIP don't merge"? I want to try and get some help to advance a couple of things in parallel. In particular, @nditada or @matiasgarciaisaia, a question for you: this line of work might result in several content files being moved and/or renamed (see: https://github.com/flattenthecurve/guide/tree/issue-213). What will be the impact of that for existing translations? I want to minimize the need to revisit all files whose content isn't affected by this.

**Comment by matiasgarciaisaia** [comment]
If a section gets moved and the filename only changes its number (ie, from `_sections/home/en/01-context/08-the-us-healthcare-system.md` to `_sections/en/01-context/15-the-us-healthcare-system.md`), the system will still be able to load the translations.

If we move them from the `en` directory, we'll probably have to fix the scripts that generate the site, export the texts and import the translations. I'll try to avoid changing the `_sections/{lang}/99-section/99-file.md` format unless there's a very good reason to do so.

**Comment by matiasgarciaisaia** [comment]
Also, remember that we can only preview PRs done _from this repo_, so instead of working in a fork, maybe import that branch in here first.

**Comment by weltyj** [comment]
> If a section gets moved and the filename only changes its number (ie, from `_sections/home/en/01-context/08-the-us-healthcare-system.md` to `_sections/en/01-context/15-the-us-healthcare-system.md`), the system will still be able to load the translations.
> 
> If we move them from the `en` directory, we'll probably have to fix the scripts that generate the site, export the texts and import the translations. I'll try to avoid changing the `_sections/{lang}/99-section/99-file.md` format unless there's a very good reason to do so.

Strong vote here for keeping a canonical structure to the file tree.   Not only does it get confusing to find the markdown or image content files, but it makes it dramatically easier to consider writing scripting processes to automate some tasks that you'd like to do on a per language basis.   My current struggle was jekyll generates the website and drops the "en" from the url path for english, but includes the two letter code for all other languages.   I worked around it but it was  frustrating and used time.

** ONE Other consideration **.  If most of the content is going to be static in terms of changing content in file names, then leaving the 2 digit prefix (11-file.md) is fine.  If you think there's going to be a lot more markdown files added, then I'd suggest moving to a (110-file.md) format now, before there's a lot more content.  That would allow a creation of a new file (115-file.md) to be inserted between 110-file.md and 120-file.md easily.   I tried the 3 digit prefix and though Jekyll warned about it, it happily did the right thing.   We (ha, me!) could automate the 3 digit prefix conversion if everyone thought it was important (I really don't know...)
